.changeset/spotty-weeks-destroy.md

```diff
@@ -0,0 +1,6 @@
+---
+"@biomejs/biome": patch
+---
+
+The [`useKeyWithClickEvents`](https://biomejs.dev/linter/rules/use-key-with-click-events/) rule has been improved with better support for ARIA roles.
+It now behaves like the original [`click-events-have-key-events`](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/click-events-have-key-events.md#rule-details) rule from `eslint-plugin-jsx-a11y`.
```

Comment:
Very few users will bother checking what the new behaviour is, and even if they do, they won't understand the improvements, especially for people that don't know anything about the original rule. My suggestion is to explicitly explain what's the improved behaviour of the rule before this paragraph.

Reply:
Updated at [87b9d86](https://github.com/biomejs/biome/pull/5507/commits/87b9d86c8823d49ed811689b7824b1d2980a194d)